server/conf/routes

```diff
@@ -63,6 +63,7 @@ POST    /admin/programs/:programId/blocks/:blockDefinitionId/questions
 POST    /admin/programs/:programId/blocks/:blockDefinitionId/questions/:questionDefinitionId/delete        controllers.admin.AdminProgramBlockQuestionsController.destroy(programId: Long, blockDefinitionId: Long, questionDefinitionId: Long)
 POST    /admin/programs/:programId/blocks/:blockDefinitionId/questions/:questionDefinitionId/setOptional   controllers.admin.AdminProgramBlockQuestionsController.setOptional(request: Request, programId: Long, blockDefinitionId: Long, questionDefinitionId: Long)
 POST    /admin/programs/:programId/blocks/:blockDefinitionId/questions/:questionDefinitionId/move          controllers.admin.AdminProgramBlockQuestionsController.move(request: Request, programId: Long, blockDefinitionId: Long, questionDefinitionId: Long)
+POST    /admin/programs/:programId/blocks/:blockDefinitionId/questions/:questionDefinitionId/setAddressCorrectionEnabled   controllers.admin.AdminProgramBlockQuestionsController.setAddressCorrectionEnabled(request: Request, programId: Long, blockDefinitionId: Long, questionDefinitionId: Long)
```

Comment:
suggestion (blocking): please align the controllers

Reply:
aligned controllers with new line